packages/components/src/components/Badge/Badge.tsx

```diff
@@ -59,14 +59,24 @@ const getIconColor = (variant: BadgeVariant, isDisabled: boolean | undefined): C
     }
 };
 
+const getPadding = (size: BadgeSize) => {
+    switch (size) {
+        case 'tiny':
+            return `0 ${spacings.xs - spacings.xxxs}px`;
```

Comment:
why not same as in `Button`?

Reply:
It's not circular with different values when we have one digit value there. And we need this for tokens in AccountsMenu. But we should definitely refactor it a bit.